pennylane/tape/tape.py

```diff
@@ -139,7 +140,8 @@ def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False):
 
     if stop_at is None:
         # by default expand all objects
-        stop_at = lambda obj: False
+        def stop_at(obj):  # pylint: disable=unused-argument
+            return False
```

Comment:
Out of curiosity: Why is this the better approach?

Reply:
`def` is preferred for its readability and flexibility (e.g.: add input types). Lambda's are useful when used only once (and thus not assigned to any variable). Moreover, when assigning a lambda to a variable, pylint raises the following error:
```
Lambda expression assigned to a variable. Define a function using the "def" keyword instead.pylint(unnecessary-lambda-assignment)
```

Reply:
Gotcha, thanks! :)